WcaOnRails/lib/paypal_interface.rb

```diff
@@ -0,0 +1,128 @@
+# frozen_string_literal: true
+
+module PaypalInterface
+  def self.generate_paypal_onboarding_link(competition_id)
+    unless Rails.env.production?
```

Comment:
I don't think you need to be this cautious.
If you think caution is warranted, please do something like `return nil if Rails.env.production?` and stop wrapping the whole method body in an "artifical" safety-check block

Reply:
"Don't need to be this cautious" as in - if we have the `Rails.env.production?` checks in the controllers, there's no way code should ever reach this interface anyway, so no need to add the env checks? 

If so, I agree

Reply:
Yes, that's what I meant.